federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -111,6 +119,25 @@ func Run(s *options.CMServer) error {
 	// Override restClientCfg qps/burst settings from flags
 	restClientCfg.QPS = s.APIServerQPS
 	restClientCfg.Burst = s.APIServerBurst
+	if err != nil {
+		glog.Fatalf("Invalid API configuration: %v", err)
+	}
+
+	kubeconfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.Kubeconfig)
```

Comment:
@madhusudancs 
The idea here is that we have 2 clientsets:
- one to talk to "kube-apiserver" of the host k8s to do leader election on multiple federation-controller-managers all registered at the host k8s
- the other one for the normal operation of federation-controller-manager, i.e., to talk to "federation-apiserver"

So,  is there a way I can pass both ip address of "kube-apiserver" and ip address of "federation-apiserver" in the flags?